uefi-mkconfig

```diff
@@ -206,6 +200,7 @@ main () {
 
 	# Add path back to the kernel image
 	for kernel_image in $kernel_images; do
+		# Variable partition_mounts in this case can't be inside of ""
```

Comment:
You can resolve this problem by using bash arrays instead of appending to a string.

Reply:
Yes I agree. This is done this way deliberately. In the future I would like to make this script more POSIX compatible so it can be used with other shells. So I try to avoid baking bash specific functions into the logic itself.